Automattic/wp-calypso
# Swap the old green for blue in illustrations used on the Plan page.

On Plans page there's a mismatch of icons - some using the new official blues, some using the old green. In this commit I'm swapping the colors, while leaving the original icons intact.

#### Testing instructions
1. Navigate to https://wordpress.com/plans/my-plan/{site}
2. Make sure none of the icons in feature cards are using green in them.
3. Check on sites with different level plans, including Jetpack sites with a plan (Personal, Premium, Professional)

**Example of a Premium plan site**

Before
![dotcom_premium](https://user-images.githubusercontent.com/4550351/57896120-75e4d480-7892-11e9-98fd-f8a58c6fe035.png)

After
![dotcom_premium_after](https://user-images.githubusercontent.com/4550351/57896278-5ac69480-7893-11e9-928a-f5349a419af9.png)

Fixes #32522
